changelog/scripts/changelog-compiler.js

```diff
@@ -0,0 +1,75 @@
+const fs = require("fs");
+const showdown = require('showdown');
+
+const converter = new showdown.Converter();
+const stdinBuffer = fs.readFileSync(0);
+const changelogAsMarkdown = stdinBuffer.toString();
+const RELEASE_REGEX = /([#]+ \[[a-zA-Z0-9.-]+].+)/g;
+const HTML_START = '<html><head><link rel="stylesheet" href="changelog.css"></head><body>';
```

Comment:
Na pewno chcesz tutaj zamykać </head> i </body>? Nie zamykasz ich przypakiem w `HTML_END`?

Reply:
masz racje, zamkniecia do wywalenia. zaraz fixne

Reply:
przeciez to zamkniecie head i otwarcie body :P dobrze jest :D